packages/gatsby-plugin-fastify/package.json

```diff
@@ -62,7 +62,7 @@
   },
   "peerDependencies": {
     "fastify": "^4.5.3",
-    "gatsby": "^4.2.0"
+    "gatsby": "^4.0.0 || ^5.0.0"
   },
   "engines": {
     "node": ">=14.15.0"
```

Comment:
Depending on if you want to change to Node 18 (required by Gatsby 5 now) this could be more strict. But this then would mean a new major version

Reply:
I have a node 18 branch in the works.  🤔 maybe I should finish that up so we can confirm testing passes. 

Also, I don't remember specifics, but I think the 4.2.0 was a bug fix or something that made this work, let's not change it at this point.

Reply:
If we'd bump to node 18 we'd need to drop Gatsby 4 and do a major

the node requirement will be checked by gatsby itself so we could theoretically keep it at >=14.15.0 without a major bump

Reply:
So Gatsby 4 doesn't support node 18. And Gatsby 5 only supports node 18?

Reply:
So other plugins I'm not to worried, but with Fastify I have to use a lot of undocumented/internal APIs. Testing is much more important here is what I'm saying. If you can't put in the time to update the integration tests and get this working we can exclude fastify from this PR and I'll handle it separately when I have time.